### ISSUE TITLE
scrollTime not updating

### ISSUE THREAD

**Issue by jonasgroendahl** [fullcalendar/fullcalendar-react#8]
Would be cool if the scrolling of the calendar would move when scrollTime is updated as prop. Not sure if it's supposed to work or not.
```
function Schedule() {
   const [scrollBy, setScrollBy] = useState('08:00:00');
       return <FullCalendar scrollTime={scrollBy} />
}
```

**Comment by arshaw** [comment]
scrollTime controls only the *initial* scroll, but doesn't change anything after that. 

It's similar in concept to how `defaultDate` controls the initial date and the methods are used to manipulate it after that. like https://fullcalendar.io/docs/react#accessing-fullcalendars-api

Luckily a `scrollToTime` method was released in v4.1.0:
https://fullcalendar.io/docs/scrollToTime

**Comment by Konstantin-Ushakov** [comment]
Could you provide an example how to do it with react? Seems like render via `FullCalendar.Calendar(Node, props)` doesn't work well.
